scripts/gcloud-sql-export.sh

```diff
@@ -51,9 +51,16 @@ function gcloud_activate_service_account() {
 function gcloud_sql_export() {
   local backup_timestamp="$1"
   gcs_backup_path="gs://${GOOGLE_SQL_BACKUP_BUCKET}/${GOOGLE_SQL_BACKUP_BUCKET_PATH}/${backup_timestamp}_${GOOGLE_SQL_INSTANCE_NAME}_${DATABASE}.gz"
-  gcloud --verbosity="${GCLOUD_VERBOSITY}" sql export sql "${GOOGLE_SQL_INSTANCE_NAME}" --database "${DATABASE}" "${gcs_backup_path}" \
-    || ( echo "Export taking longer than expected - waiting another ${GCLOUD_WAIT_TIMEOUT} seconds." ; \
-      gcloud --verbosity="${GCLOUD_VERBOSITY}" sql operations wait --timeout "${GCLOUD_WAIT_TIMEOUT}" --quiet $(gcloud sql operations list --instance="${GOOGLE_SQL_INSTANCE_NAME}" --filter='status=RUNNING' --format="value(NAME)") )
+  gcloud --verbosity="${GCLOUD_VERBOSITY}" sql export sql "${GOOGLE_SQL_INSTANCE_NAME}" --database "${DATABASE}" "${gcs_backup_path}" --async
+  echo "Export started, waiting ${GCLOUD_WAIT_TIMEOUT} seconds for it to complete."
+  if ! gcloud --verbosity="${GCLOUD_VERBOSITY}" sql operations wait --timeout "${GCLOUD_WAIT_TIMEOUT}" --quiet "$(gcloud sql operations list --instance="${GOOGLE_SQL_INSTANCE_NAME}" --filter='status=RUNNING' --format="value(NAME)")"; then
```

Comment:
should we loop here and check a couple of times ?

Reply:
I dont think so, if I specify a timeout to the container I would assume that it would wait that long and then time out, rather than retrying multiple times with that same timeout.